mnefun/_mnefun.py

```diff
@@ -364,6 +367,11 @@ def do_processing(p, fetch_raw=False, do_score=False, push_raw=False,
              'Status',
              ]
     score_fun = p.score if p.score is not None else default_score
+    if len(inspect.getargspec(score_fun).args) == 2:
+        score_fun_two = score_fun
+
+        def score_fun(p, subjects, run_indices):
```

Comment:
If you're using the built-in simple scoring, it works. If you wrote your own scoring script, you need to modify it to take in the `run_indices` as a third argument, and deal with it yourself. But the built-in scoring functions shows how it can be done.